src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -2845,19 +2861,8 @@ bool Eltwise::canFuse(const NodePtr& node) const {
     if (!mayiuse(x64::sse41) || getInputShapeAtPort(0).getRank() > MAX_ELTWISE_DIM_RANK)
         return false;
 
-    // TODO: supported only via reference executor
-    if (one_of(getAlgorithm(),
```

Comment:
We still need to keep the condition. 
Bitwise operations can work correclty only with integer precisions. Fusion of any other elementwise operation (like Add) will lead to the fact whole chain of operation will be computed in FP32 precision.

Reply:
done, thanks